src/Tizen.WebView/Tizen.WebView/WebView.cs

```diff
@@ -77,6 +79,18 @@ public class WebView: EvasObject
         /// <since_tizen> 4 </since_tizen>
         public event EventHandler<SmartCallbackArgs> UrlChanged;
 
+        /// <summary>
+        /// Event that occurs when the title of the main frame is changed.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public event EventHandler<ContextMenuArgs> ContextMenuCustomize;
```

Comment:
Plz update summary properly.

Reply:
First, you need to update description
/// Event that occurs when the context menu customized.
It is wrong

And to provide a way to customize the context menu.
I recommand to use callback instead of event

  public  delegate ContextMenuCustomize(ContextMenu menu);
  public void SetContextMenuCustomizeDelegate(ContextMenuCustomize delegate);